index.html

```diff
@@ -11,22 +11,214 @@
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+        </div>
+            </section>
+            <section class="AboutMeHeader">
+                <div class="resume">
+                <h3 class="me">About me</h3>
+                <p class="help">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                   don’t hestiate to contact me.</p>
+                <button class="projects" type="button">Get My Resume</button>
+                </div>
+                <br>
+                <div class="planholder">
+                    <div class="box1">
+                        <div class="skills">
+                            <img src="images/icon-languages.svg" alt="icon languages" class="icon">
+                            <h2 class="skills">Languages</h2>
+                            <ul class="language">
+                            
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">css</button></li>
+                            <li><button type="button">Java Script</button></li>
+                            <li><button type="button">HTML</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box2">
+                        <div class="skills">
+                            <img src="images/icon-frameworks.svg" alt="icon Frameworks" class="icon">
+                            <h2 class="skills">Frameworks</h2>
+                            
+                            <ul class="language">
+                            <li><button type="button">Bootstrap</button></li>
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">RSpec</button></li>
+                            <li><button type="button">Capybara</button></li>
+                            <li><button type="button">Selenium</button></li>
+                        </ul>
```

Comment:
- [ ] (Optional) It would be nice to add some padding to the badges used here and make the `color` the same as that of the design. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172945328-0bab37cc-1d8e-4896-9e25-bad0a1c6160d.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172945389-7ae56006-040e-4b07-a110-2fa23af294d2.png)